webui/react/src/components/Link.tsx

```diff
@@ -10,38 +9,38 @@ export interface Props {
   disabled?: boolean;
   inherit?: boolean;
   isButton?: boolean;
-  noProxy?: boolean;
+  external?: boolean;
   path?: string;
   popout?: boolean;
   onClick?: MouseEventHandler;
 }
 
 const Link: React.FC<Props> = ({
-  noProxy,
+  external,
   path = '#',
   popout,
   onClick,
   ...props
 }: PropsWithChildren<Props>) => {
   const classes = [ css.base ];
   const rel = windowOpenFeatures.join(' ');
-  const linkPath = noProxy ? serverAddress(true, path) : path;
 
   if (props.className) classes.push(props.className);
   if (!props.disabled) classes.push(css.link);
   if (props.inherit) classes.push(css.inherit);
   if (props.isButton) classes.push('ant-btn');
 
+  path = (external ? '' : process.env.PUBLIC_URL) + path;
```

Comment:
**nit:** I think in practice it's better to avoid modifying existing props or function params for the functional approach. ```
const linkPath = (external ? '' : process.env.PUBLIC_URL) + path;
```
or if you have a better name than `linkPath`